nova/tests/unit/virt/vmwareapi/test_vmops.py

```diff
@@ -79,7 +79,8 @@ def setUp(self):
 
         self._virtapi = mock.Mock()
         self._image_id = nova.tests.unit.image.fake.get_valid_image_id()
-        fake_ds_ref = vmwareapi_fake.ManagedObjectReference(value='fake-ds')
+        fake_ds_ref = vmwareapi_fake.ManagedObjectReference(name='fake-ds',
+                                                            value='Datastore')
```

Comment:
`name` and `value` needs to be switched. The `ManagedObjectReference` creates the `type` attribute out of the `name` and thus the `name` should be `'Datastore'`.

Reply:
Done